TestVectorsAwsCryptographicMaterialProviders/runtimes/python/fuzz_generator.py

```diff
@@ -0,0 +1,262 @@
+#!/usr/bin/env python3
+"""
+Fuzz Testing Generator for AWS Cryptographic Material Providers Library
+
+This script generates fuzzed test vectors with a focused approach:
+1. Fuzz ONLY the 3 high-priority encryption context fields (infinite possibilities)
```

Comment:
unicode characters are not infinite but its bounded between two codepoint ranges. Quick google search tells me "Unicode comprises 1,114,112 code points in the range 0hex to 10FFFFhex"